src/pages/TodayPage/Components/CreateReminder.tsx

```diff
@@ -0,0 +1,184 @@
+import {
+  Box,
+  Button,
+  Flex,
+  Input,
+  Spacer,
+  Text,
+  useMediaQuery,
+} from "@chakra-ui/react";
+import { useState } from "react";
+import {
+  AiFillFolderOpen,
+  AiOutlineCalendar,
+  AiOutlinePlusCircle,
+} from "react-icons/ai";
+import Centere from "./Center";
+function CreateReminder() {
+  const [isAddTaskOpen, setIsAddTaskOpen] = useState(false);
+  const [taskName, setTaskName] = useState("");
+  const [isLargerThan800] = useMediaQuery("(min-width: 800px)");
+  return (
+    <>
+      {!isAddTaskOpen && (
+        <>
+          <Flex
+            w={isLargerThan800 ? "60%" : "80%"}
+            mt='2'
+            alignItems='center'
+            cursor='pointer'
+          >
+            <Text
+              _hover={{ bg: "red", color: "white" }}
+              onClick={() => setIsAddTaskOpen(!isAddTaskOpen)}
+              borderRadius='100%'
+              color='red'
+            >
+              <AiOutlinePlusCircle />
+            </Text>
+            <Flex
+              color='gray'
+              ml='2'
+              _hover={{ color: "red" }}
+              onClick={() => setIsAddTaskOpen(!isAddTaskOpen)}
+            >
+              Görev Ekle
+            </Flex>
+          </Flex>
+          <Centere />
+        </>
+      )}
+      {isAddTaskOpen && (
+        <Flex direction='column' w={isLargerThan800 ? "60%" : "80%"} mt='4'>
+          <Box
+            display='flex'
+            alignItems='center'
+            justifyContent='center'
+            flexDirection='column'
+          >
+            <Flex
+              border='1px'
+              borderColor='gray.300'
+              h='120px'
+              w='100%'
+              borderRadius='xl'
+            >
+              <Box w='200px'>
+                <Input
+                  value={taskName}
+                  onChange={(e) => setTaskName(e.target.value)}
+                  ml='3'
+                  mt='3'
+                  variant='unstyled'
+                  placeholder='Görev ismi'
+                  _placeholder={{ opacity: 1, color: "gray.500" }}
+                />
+                <Input
+                  mt='2'
+                  ml='3'
+                  variant='unstyled'
+                  placeholder='Açıklama'
+                  _placeholder={{ opacity: 1, color: "gray.500" }}
+                />
+                <Flex direction='row' ml='3' mt='2'>
```

Comment:
Flex direction default value is already "Row". No need to rewrite

Reply:
thank you for the fix